packages/lodestar/src/network/network.ts

```diff
@@ -78,13 +78,15 @@ export class Libp2pNetwork extends (EventEmitter as {new (): NetworkEventEmitter
     await this.reqResp.start();
     const enr = this.getEnr();
     await this.metadata.start(enr!);
+    await this.gossip.pubsub.start();
```

Comment:
How about having `pubsub` as part of network instead of gossip? So `pubsub` could be created/started/stopped inside network, gossip just use it. We have `pubsub` as part of `IGossipModules` anyway.

So this could become: `this.pubsub.start()`

Reply:
:+1: sounds good

Reply:
In fact, libp2p already comes with a pubsub attached! We might look into just using that one in a later PR